File: csharp/HearthstoneReplays/ReplayData/Meta/Info.cs
#region

using System.Xml.Serialization;

#endregion

namespace HearthstoneReplays.ReplayData.Meta
{
	public class Info : GameData
	{
		[XmlAttribute("info")]
		public int Index { get; set; }

		[XmlAttribute("id")]
		public int Id { get; set; }
	}
}

File: java/src/main/java/info/hearthsim/hsreplay/parser/replaydata/meta/Info.java
package info.hearthsim.hsreplay.parser.replaydata.meta;

import info.hearthsim.hsreplay.parser.replaydata.GameData;

import javax.xml.bind.annotation.XmlAccessOrder;
import javax.xml.bind.annotation.XmlAccessType;
import javax.xml.bind.annotation.XmlAccessorOrder;
import javax.xml.bind.annotation.XmlAccessorType;
import javax.xml.bind.annotation.XmlAttribute;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@XmlAccessorType(XmlAccessType.FIELD)
@XmlAccessorOrder(XmlAccessOrder.ALPHABETICAL)
@ToString
public class Info extends GameData {

	@XmlAttribute(name = "index")
	private int index;

	@XmlAttribute(name = "id")
	private int id;
}
